### DISCUSSION TITLE
Align Tools in OpenToonz

### DISCUSSION THREAD

**Discussion by erayzesen** [opentoonz/opentoonz#6359] [Ideas]
### Is your feature request related to a problem? Please describe

It is a feature that I have always missed, especially when using vector levels in Opentoonz. It would be very helpful for speeding up work if there was an align tool like in Inkscape.

### Describe the solution you'd like

Even if the functionality in Inkscape were implemented in the most basic way, it would be great. Below, I am providing a screenshot of the application in Inkscape.

![Screenshot_2023-10-23_00-30-11](https://github.com/opentoonz/opentoonz/assets/65820625/d0d7615e-dbce-4040-9b4f-24e2ba418a10)

![Screenshot_2023-10-23_00-29-31](https://github.com/opentoonz/opentoonz/assets/65820625/90a730aa-2829-4cd3-a8ea-f6fb20e9a301)

**Comment by RodneyBaker** [comment]
This may not be quite what you are after @erayzesen but John (manongjohn) is implementing some vector alignment tools in Tahoma2D.  You may wish to check that out:  https://github.com/tahoma2d/tahoma2d/pull/1275

It takes time but eventually tools from OT and T2D tend to get ported to each other.

You may want to provide feedback on that effort as I'm sure John will appreciate that.

There is an installer as well as a portable build you can check out here:  
https://github.com/tahoma2d/tahoma2d/actions/runs/6832386848?pr=1275#artifacts

**Comment by erayzesen** [comment]
Thank you @RodneyBaker , it looks like a part of what I want.  I hope this side can be improved.

**Comment by RodneyBaker** [comment]
> I hope this side can be improved.

I'm not only sure that it can but that it will.

**Comment by RodneyBaker** [comment]
Update:  John has further updated his implementation of vector alignment and distribution tools in Tahoma2D.
Check out the appropriate release of T2D for more information and to explore the feature.

John has also create a dedicated window/panel for the tools:
![image](https://github.com/opentoonz/opentoonz/assets/2576432/cfd33b31-1f6a-40d1-b659-eabb15eb9cd6)
